lib/lock-file-parser.ts

```diff
@@ -7,11 +8,15 @@ export function packageSpecsFrom(
   try {
     lockFile = toml.parse(lockFileContents) as unknown as PoetryLockFile;
   } catch {
-    throw new LockFileNotValid();
+    throw new OpenSourceEcosystems.UnparseableLockFileError(
+      'The poetry.lock file is not parsable.',
+    );
```

Comment:
Can we add the error to the error args? This gives zero information in the logs.